TypeCobol/AntlrGrammar/CobolWords.g4

```diff
@@ -563,7 +563,24 @@ tokens
     PRIVATE,
     IN_OUT,
 	STRICT,
-	QuestionMark
+	QuestionMark,
```

Comment:
Token should be written in upper case.
I made the mistake with QuestionMark.

We should fix all tokens in this PR.